validator/validator-main.protoascii

```diff
@@ -4486,6 +4486,20 @@ attr_lists: {
     value_casei: "parallax"
     requires_extension: "amp-fx-collection"
   }
+  # amp-lightbox-gallery specific attributes, see
+  # https://www.ampproject.org/docs/reference/components/amp-lightbox-gallery
+  attrs: {
```

Comment:
to @honeybadgerdontcare points, yeah we don't want these on everything. Maybe makes sense to create an `attr_lists` for `lightboxable elements` (similar to how `extended-amp-global` works for common attributes like `layout`)  and add that to only components we currently support.

Reply:
+1. 
Refactored this to produce a `lightboxable-elements` `attr_list`, stuck on some whitespace validation issue again.